polaris/step.py

```diff
@@ -461,6 +483,37 @@ def add_dependency(self, step, name=None):
         target = f'{step.path}/step_after_run.pickle'
         self.add_input_file(filename=filename, work_dir_target=target)
 
+    def validate_baselines(self):
+        """
+        Compare variables between output files in this step and in the same
+        step from a baseline run if one was provided.
+
+        Returns
+        -------
+        compared : bool
+            Whether a baseline comparison was performed
+
+        success : bool
+            Whether the outputs were successfully validated against a baseline
+        """
+        if self.work_dir is None:
+            raise ValueError('Baselines cannot be validated before the work '
+                             'directory has been set')
```

Comment:
```suggestion
            raise ValueError('The work directory must be set before the step '
                             'outputs can be validated against baselines.')
```
Maybe this is clearer?